licenses/urls.py

```diff
@@ -136,33 +143,32 @@ def to_url(self, value):
 
 # DEEDS
 urlpatterns = [
-    path(
-        "license/", license_detail, name="license_detail"
-    ),
-    path(
-        "sampling/", sampling_detail, name="sampling_detail"
-    ),
-    path(
-        "deed/", deed_detail, name="deed_detail"
-    ),
-    path(
-        "<code:license_code>/<version:version>",
+    # New templates to be used
+    path("license/", license_detail, name="license_detail"),
+    path("sampling/", sampling_detail, name="sampling_detail"),
+    path("deed/", deed_detail, name="deed_detail"),
```

Comment:
It looks like we could change `path` to `distill_path` here and add a no-op distill_func and we'd get our test pages included in the static site.

Reply:
done